source/agora/node/Ledger.d

```diff
@@ -557,7 +557,11 @@ public class Ledger
                 assert(0);
             }
         }
-        this.enroll_man.updateValidatorIndexMaps(block.header.height + 1);
+
+        const nheight = block.header.height + 1;
```

Comment:
nheight -> next

Reply:
That one was actually on purpose (nheight => next height), but I guess next is better.

Reply:
I thought maybe that was the case but think next is still better.